core/domain/feedback_services.py

```diff
@@ -713,17 +801,31 @@ def get_exp_thread_summaries(user_id, thread_ids):
     ]
     exp_model_ids = [model.entity_id for model in exp_thread_models]
 
-    exp_thread_user_models, exp_models = (
-        datastore_services.fetch_multiple_entities_by_ids_and_models([
-            ('GeneralFeedbackThreadUserModel', exp_thread_user_model_ids),
-            ('ExplorationModel', exp_model_ids),
-        ]))
+    # Here, cast is used to narrow down the return type of following method from
+    # List[Optional[Model]] to List[Optional[exp_models.ExplorationModel]].
+    exp_thread_user_models, exp_models_list = (
```

Comment:
```suggestion
    exp_thread_user_models, exp_models = (
```

Reply:
This is done on purpose!!
If we see line `40` then there we have the same name.

Reply:
Line `40` of feedback_services.py

Reply:
I see. Then maybe `exploration_models` to be consistent with `exp_thread_user_models`.

Reply:
Done!